campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -266,6 +258,15 @@
                 var btnSelected;
                 var amount;
                 var presets = '{{donationAmountFirst}},{{donationAmountSecond}},{{donationAmountThird}},{{donationAmountFourth}}';
+                var numberFormat = new Intl.NumberFormat('{{locale}}', {
```

Comment:
`locale` needs a default value, especially because we already run the snippet and any changes must be backwards compatible